.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "plugins": [
-    "transform-class-properties",
-    "transform-object-assign",
-    "transform-object-rest-spread"
-  ],
-  "presets": ["es2015", "react"]
+  "presets": ["env", "react"]
```

Comment:
I followed the documentation https://github.com/babel/babel-preset-env#usage

But I am not sure if we should use other configs. We can improve that in the future, what do you think?